docs/execute-methods.md

```diff
@@ -269,6 +269,20 @@ isRoot | boolean | no | This option defines where the certificate should be inst
 
 The content of the generated .mobileconfig file as base64-encoded string. This config might be useful for debugging purposes. If the certificate has been successfully set via CLI then nothing is returned.
 
+### mobile: removeCertificate
+
+Removes installed certificate for real devices only if [py-ios-device](https://github.com/YueChen-C/py-ios-device) tool is available on the server machine since driver version 4.19.2.
```

Comment:
What happens if the certificate with the given name does not exist/is already removed?

Reply:
Right now py-ios-device returns `{'Status': 'Acknowledged'}` if the certificate is removed or `None` if there was no certificate removed. I chose to not alter this behavior, but if that behavior is unclear, we could catch the `None` and return as `{'Status': 'Failed'}` or something similar

Reply:
Updated documentation